vocab/credentials/v2/vocabulary.yml

```diff
@@ -33,7 +33,11 @@ class:
     label: JSON schema validator 2023
     defined_by: https://www.w3.org/TR/vc-json-schema/#jsonschema2023
```

Comment:
can remove year here too?

Reply:
I am not sure what is going on, ie, why the preview in the PR does not show up properly. If you look at the file directly at https://github.com/w3c/vc-data-model/blob/iherman-adding-schema-class/vocab/credentials/v2/vocabulary.yml the date has been removed already...